cirq-web/cirq_ts/dist/bloch_sphere.bundle.js

```diff
@@ -0,0 +1,2 @@
+/*! For license information please see bloch_sphere.bundle.js.LICENSE.txt */
```

Comment:
This change reminded me: as the bundles are generated, we should add a check that ensures that the generated files are up to date. I.e. 

1. `check/npm run build`
2. check if there are any changes in the dist directory - if there are, fail with a nice error message that tells the developer what to run and commit.

Reply:
So basically this is just making sure that people commit their changed bundle.js files?

Reply:
Done

Reply:
I don't see it added to the ci.yml - am I missing something?